src/windows/MediaProxy.js

```diff
@@ -226,6 +301,10 @@ module.exports = {
         try {
             if (thisM.node) {
                 thisM.node.onloadedmetadata = null;
+                // Unsubscribing as the media object is being released
+                thisM.node.onerror = null;
+                // Needed to avoid "0x80070005 - JavaScript runtime error: Access is denied." on copyAsync
```

Comment:
Can you clarify here why this error is occurring?  Is it because there is a sharing violation (i.e., the file is already open when trying to overwrite it)?